packages/js/src/helpers/checkForMetaboxWarnings.js

```diff
@@ -0,0 +1,13 @@
+import { select } from "@wordpress/data";
+/**
+ * Checks if there are any warnings in the metabox.
+ *
+ * @returns {boolean} Whether there are any warnings.
+ */
+export const checkForMetaboxWarnings = () => {
+	const isPremium = select( "yoast-seo/editor" ).getIsPremium();
+	const warningsFree = select( "yoast-seo/editor" ).getWarningMessage();
+	const warningsPremium = isPremium ? select( "yoast-seo-premium/editor" )?.getMetaboxWarning() ?? [] : false;
```

Comment:
Does a .length on false not give a notice?

Reply:
Right, fixed.